tests/ClientBuilderTest.php

```diff
@@ -82,7 +82,7 @@ public function testSetTransport(): void
         /** @var TransportInterface|MockObject $transport */
         $transport = $this->createMock(TransportInterface::class);
 
-        $clientBuilder = new ClientBuilder(['dsn' => 'http://public:secret@example.com/sentry/1']);
+        $clientBuilder = new ClientBuilder(new Options(['dsn' => 'http://public:secret@example.com/sentry/1']));
```

Comment:
To simplify the tests I would use the `create` method so that we avoid creating the `Options` object everywhere